lib/nostrum/shard/event.ex

```diff
@@ -32,10 +32,7 @@ defmodule Nostrum.Shard.Event do
 
   def handle(:heartbeat_ack, _payload, state) do
     Logger.debug "HEARTBEAT_ACK"
-    heartbeat_intervals = state.heartbeat_intervals
-    |> List.delete_at(-1)
-    |> List.insert_at(0, Util.now() - state.last_heartbeat)
-    %{state | heartbeat_intervals: heartbeat_intervals}
```

Comment:
We were keeping track of this to supply some form of gateway latency to the user. Do you still think it's a good idea to remove it? I'm not too keen on what would be useful to supply to the user from the shard state, maybe we should have a discussion on this.